src/urllib3/response.py

```diff
@@ -452,6 +453,7 @@ def __init__(
 
         self._pool = pool
         self._connection = connection
+        self.length = length
```

Comment:
I feel like this length should interact someway with the init_length thing, but I'm not sure how.

Reply:
`length` is defined in http.client as "Number of bytes left in the response" so... looks like they're the same value? Let's stick with our existing property then and remove `length`.

Reply:
and here I was thinking "total" content-length... 😊 

done